frontend/lib/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -39,82 +33,44 @@ const dummyGraphviz = graphviz
 dummyGraphviz // eslint-disable-line @typescript-eslint/no-unused-expressions
```

Comment:
Hey @snehankekre Do you think we can remove the above code? It looks like it works in my testing.

The way I am looking at this, the variable declaration/usage does not...actually call anything. If anything, it's ensuring the bundler (aka Webpack is including the d3-graphviz library). Since it's in the React component, it seems like the code is being included (and tests seem to show it appearing), it seems like it should work.

Reply:
Removed on 51c99f5. I kept it in as it used to exist and since I wasn't sure of the implications of removing it.